enterprise/coderd/provisionerdaemons.go

```diff
@@ -117,6 +117,13 @@ func (p *provisionerDaemonAuth) authorize(r *http.Request, tags map[string]strin
 	if p.psk != "" {
 		psk := r.Header.Get(codersdk.ProvisionerDaemonPSK)
 		if subtle.ConstantTimeCompare([]byte(p.psk), []byte(psk)) == 1 {
+			if len(tags) == 0 {
+				// Directly scope to organization if no tags are provided.
+				// MutateTags is only meant for scoping based on users.
+				tags = map[string]string{
+					provisionerdserver.TagScope: provisionerdserver.ScopeOrganization,
+				}
+			}
```

Comment:
instead, I think we should just unconditionally set 

`tags[provisionerdserver.TagScope] = provisionerdserver.ScopeOrganization`

Otherwise you could still use the PSK to grab user local jobs by setting `scope=user, owner=<userid>`

Reply:
In this case, it might also be worth doing a check in the CLI provisionerd command to error if you use PSK auth and set `scope=user`.